.github/ISSUE_TEMPLATE/triage_membership.md

```diff
@@ -1,12 +1,13 @@
 ---
 name: Python Triage Membership
 about: Request by a core dev for someone to join the Python Triage team
+title: "Request for Python triage membership: name"
 ---
 
 <!--
 Core developers can create this issue to nominate someone to the Python Triage team.
 
-Existing bug triagers on bpo and other Python contributors with elevated privileges can create this issue
+Existing bug triagers and other Python contributors with elevated privileges can create this issue
```

Comment:
I'm not sure this sentence still makes sense since it appears to refer to bpo triagers. There may still be people who have bpo triage permissions but who are not triagers on GitHub, but I'm not sure we should treat them specially at this point.

Reply:
Yeah, let's just remove this sentence.